docs/_static/version_switcher.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "name": "v2.8.0 (devel)",
```

Comment:
NITPIC, I feel just calling this "v2.8.0 (dev)" may be better if just trying to shorten the word develop